src/components/Card.jsx

```diff
@@ -4,8 +4,13 @@ import { useProduct } from "../services/product_api";
 import { useWishlist } from "../context/wishlistcontext";
 import { useFilter } from "../context/filtercontext";
 import { sortedData } from "../context/filtercontext";
```

Comment:
if both imports come from the same file you can use  `import {useFilter, sortedData } from "../context/filtercontext"`